src/lighter/main.py

```diff
@@ -162,6 +162,10 @@ def parse_service(filename, targetdir=None, verifySecrets=False):
     config['labels'] = config.get('labels', {})
     config['labels']['com.meltwater.lighter.checksum'] = checksum
 
+    # Include a docker label to sort on
+    config['container']['docker']['parameters'] = config['container']['docker'].get('parameters', [])
```

Comment:
This would fail if the task isn't a docker one, perhaps wrap with an `if util.rget(config, 'container', 'docker'):` and add a unit test where container/docker/ isn't present?